homeassistant/components/ness_alarm/__init__.py

```diff
@@ -109,6 +113,14 @@ async def _close(event):
 
     hass.bus.async_listen_once(EVENT_HOMEASSISTANT_STOP, _close)
 
+    async def _started(event):
+        # Force update for current arming status and current zone states (once Home Assistant has finished loading required sensors and panel)
+        _LOGGER.debug("invoking client keepalive() & update()")
+        hass.loop.create_task(client.keepalive())
```

Comment:
Side note: Please use `asyncio.create_task` instead. The loop interface for this feature is legacy.

Reply:
Are these tasks cleaned up automatically?